alonzo/test/lib/Test/Cardano/Ledger/Alonzo/Examples/Consensus.hs

```diff
@@ -0,0 +1,174 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Test.Cardano.Ledger.Alonzo.Examples.Consensus where
+
+import Cardano.Ledger.Alonzo (AlonzoEra)
+import Cardano.Ledger.Alonzo.Data (AuxiliaryData (..), AuxiliaryDataHash (..), Data (..), hashData)
+import Cardano.Ledger.Alonzo.PParams (PParams' (..), emptyPParams, emptyPParamsUpdate)
+import Cardano.Ledger.Alonzo.Scripts (ExUnits (..), Script (..), alwaysFails, alwaysSucceeds)
+import qualified Cardano.Ledger.Alonzo.Scripts as Tag (Tag (..))
+import Cardano.Ledger.Alonzo.Tx (IsValidating (..), ValidatedTx (..))
+import Cardano.Ledger.Alonzo.TxBody (TxBody (..), TxOut (..))
+import Cardano.Ledger.Alonzo.TxWitness (RdmrPtr (..), Redeemers (..), TxDats (..), TxWitness (..))
+import Cardano.Ledger.BaseTypes (StrictMaybe (..))
+import Cardano.Ledger.Coin (Coin (..))
+import Cardano.Ledger.Core (TxBody)
+import Cardano.Ledger.Crypto (Crypto, StandardCrypto)
+import Cardano.Ledger.Era (ValidateScript (hashScript))
+import Cardano.Ledger.Keys (asWitness)
+import Cardano.Ledger.Mary.Value (AssetName (..), PolicyID (..), Value (..))
+import Cardano.Ledger.SafeHash (hashAnnotated)
+import Cardano.Ledger.ShelleyMA.Timelocks (Timelock (..), ValidityInterval (..))
+import Cardano.Slotting.Slot (EpochNo (..), SlotNo (..))
+import Data.Default.Class (def)
+import qualified Data.Map.Strict as Map
+import Data.Proxy (Proxy (..))
+import qualified Data.Sequence.Strict as StrictSeq
+import qualified Data.Set as Set
+import qualified PlutusTx as Plutus
+import Shelley.Spec.Ledger.API
+  ( ApplyTxError (..),
+    Credential (..),
+    Network (..),
+    NewEpochState (..),
+    ProposedPPUpdates (..),
+    RewardAcnt (..),
+    TxId (..),
+    TxIn (..),
+    Update (..),
+    Wdrl (..),
+  )
+import Shelley.Spec.Ledger.STS.Delegs (DelegsPredicateFailure (..))
+import Shelley.Spec.Ledger.STS.Ledger (LedgerPredicateFailure (..))
+import Shelley.Spec.Ledger.Tx (Tx (..))
+import Shelley.Spec.Ledger.UTxO (makeWitnessesVKey)
+import qualified Test.Shelley.Spec.Ledger.Examples.Consensus as SLE
+import Test.Shelley.Spec.Ledger.Orphans ()
+import Test.Shelley.Spec.Ledger.Utils (mkAddr)
+
+type StandardAlonzo = AlonzoEra StandardCrypto
+
+-- | ShelleyLedgerExamples for Alonzo era
+ledgerExamplesAlonzo :: SLE.ShelleyLedgerExamples StandardAlonzo
+ledgerExamplesAlonzo =
+  SLE.ShelleyLedgerExamples
+    { SLE.sleBlock = SLE.exampleShelleyLedgerBlock exampleTransactionInBloc,
```

Comment:
I was tempted to overhaul `exampleShelleyLedgerBlock` to allow for multiple transactions in the example block. Especially since there are some subtleties as to how transactions are packed into a block. But I wasn't sure how soon you wanted these examples. I could even do that in a follow up PR if you'd like.

Reply:
Yes, a follow-up PR would be nice. It'd let us choose how big a bite we want to chew at once :)